### PR TITLE
OCPBUGS#14864 4.10: Add known issue Selinux relabeling

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.10 only

Issue: https://issues.redhat.com/browse/OCPBUGS-14864

Link to docs preview: https://file.rdu.redhat.com/lpettyjo/OCPBUGS-14864_4_10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-known-issues

QE review:
- [x] QE has approved this change.

Additional information:

PTAL: @gcharot, @chao007, @sferich888, @xltian, @mwerner2113, @gnufied

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -2653,6 +2653,8 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
 
 * If a cluster that was deployed through ZTP has policies that do not become compliant, and no `ClusterGroupUpdates` object is present, you must restart the {cgu-operator} pods. Restarting {cgu-operator} creates the proper `ClusterGroupUpdates` object, which enforces the policy compliance. (link:https://issues.redhat.com/browse/OCPBUGS-4065[*OCPBUGS-4065*])
 
+* Currently, when using a persistent volume (PV) that contains a very high file count, the pod may not start or takes an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-10-release-notes.adoc:2656]:
```suggestion
* Currently, when using a persistent volume (PV) that contains a very high file count, the pod may not start or may take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
```

**Review by jeana-redhat — COMMENTED** [review]
Same as 4.12

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by gnufied** [comment]
lgtm

**Comment by gcharot** [comment]
lgtm

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (chao007) [reply release_notes/ocp-4-10-release-notes.adoc:2656]: I guess the reason is the title is high file counts in https://access.redhat.com/solutions/6221251

**Comment by xltian** [comment]
lgtm

**Comment by gnufied** [comment]
lgtm

**Comment by kalexand-rh** [comment]
LGTM

**Pull request merged**